rules/REQUEST-903.9006-XENFORO-EXCLUSION-RULES.conf

```diff
@@ -80,8 +80,9 @@ SecRule REQUEST_FILENAME "@rx /(?:conversations|(?:conversations|forums|threads)
 # POST /xf/threads/thread-title.12345/add-reply
 # POST /xf/threads/thread-title.12345/reply-preview
 # POST /xf/forums/forum-title.12345/post-thread
+# POST /xf/forums/blogs/post-thread
 # POST /xf/forums/forum-title.12345/thread-preview
-SecRule REQUEST_FILENAME "@rx /(?:conversations/add(?:-preview)?|conversations/messages/\d+/edit|posts/\d+/(?:edit|preview)|(?:conversations|threads)/.*\.\d+/(?:add-reply|reply-preview)|forums/.*\.\d+/(?:post-thread|thread-preview))$" \
+SecRule REQUEST_FILENAME "@rx /(?:conversations/add(?:-preview)?|conversations/messages/\d+/edit|posts/\d+/(?:edit|preview)|(?:conversations|threads)/.*\.\d+/(?:add-reply|reply-preview)|forums/.*/(?:post-thread|thread-preview))$" \
```

Comment:
From the regex POV, using `[^/]+` would be better.

Reply:
I'm not 100% sure that there may not be a `/` here. I had already made an assumption based on traffic and that didn't hold up. So I'm tempted to leave this for now.

Reply:
I was using the paths in the comments on this rule, but if you are not sure we can ignore this.